### PR TITLE
Fixing Accessibility issues

### PR DESCRIPTION
**Pull request by jhkimnew**
Fixed Truncation issues:
https://github.com/microsoft/IIS.WebManager/issues/82
https://github.com/microsoft/IIS.WebManager/issues/101

### CODE REVIEW

File: src/themes/main.css [hunk 1]
```diff
@@ -549,12 +549,9 @@ select[enabled]:not(.inactive) {
 }
 
 .center {
-    position:fixed;
+    position: relative;;
```
Comment (jhkimnew) [inline src/themes/main.css]: Consider using different class name for the problematic page.

File: src/app/main/get.component.ts [hunk 2]
```diff
@@ -43,7 +43,11 @@ import { HttpFacade } from 'common/http-facade';
         </div>
     `,
     styles: [`
-        .center {
+        .center-page {
```
Comment (shhsu) [inline src/app/main/get.component.ts]: refactor to main.css

File: src/app/common/tabs.component.ts [hunk 3]
```diff
@@ -286,6 +301,34 @@ export class TabsComponent implements OnDestroy, AfterViewInit {
...
+            this.showMenu(false);
+            return;
+        }
+        this.showMenu(true);
+    }
+
+    private onClickToggleMenu() {
+        this._toggleOnProcessing = true;
+
+        window.setTimeout(() => {
```
Comment (shhsu) [inline src/app/common/tabs.component.ts]: to prevent double trigger, try to pass the event in the click and call `e.preventDefault()`

Comment (shhsu) [reply src/app/common/tabs.component.ts]: And also check for `e.defaultPrevented` on toggle menu

File: src/app/common/tabs.component.ts [hunk 4]
```diff
@@ -286,6 +301,34 @@ export class TabsComponent implements OnDestroy, AfterViewInit {
         this._menuOn = (show == null) ? true : show;
     }
 
+    private toggleMenu() {
```
Comment (shhsu) [inline src/app/common/tabs.component.ts]: fix `showMenu`

File: src/app/common/tabs.component.ts (hunk 3 above)
Comment (jhkimnew) [reply src/app/common/tabs.component.ts]: preventDefault() does not work for this case. I will open a new issue to investigate the issue that both click and enter key event happens even though I clicked only enter key.

File: src/app/common/tabs.component.ts [hunk 5]
```diff
@@ -190,10 +183,29 @@ import { SectionHelper } from './section.helper';
                     <div class="hider background-normal"></div>
                 </div>
             </div>
-            <div class='menu-btn color-active background-normal' #menuBtn (click)="showMenu(true)"><span class="border-active hover-active color-normal" [class.background-active]="_menuOn"><i class="fa fa-ellipsis-h"></i></span></div>
+            <div class='menu-btn color-active background-normal' #menuBtn>
+                <span
+                    tabindex="0"
+                    (keyup.space)="onToggleMenu($event)"
```
Comment (shhsu) [inline src/app/common/tabs.component.ts]: reminder: remove

**Review by shhsu — APPROVED** [review]

**Pull request merged**